### ISSUE TITLE
No response from API using node.js 7.7.2 

### ISSUE THREAD

**Issue by phlummox** [xBrowserSync/API#2]
Hi, just following up from the previous issue I logged, for whenever you have a chance.

I created an instance of the API server in a Docker container, using the Dockerfile attached: [Dockerfile.txt](https://github.com/xBrowserSync/API/files/1688125/Dockerfile.txt)

This installs mongodb, node.js 7.7.2 and the API server, configures mongodb, and creates a small launch script. When I run the API server using it, nothing gets output to the `/var/log/xBrowserSync_api.log` log file, and any attempts to connect to `https://localhost:8080/` just time out.

Unfortunately I'm not able at the moment to investigate further, but let me know if there's any other information that would be useful in resolving the issue.

EDIT: Ah - I didn't realize that by default, the API runs over plain HTTP, rather than TLS.  I can report that I now have the server running. It'd be great if the README could include any info on configuring it to use TLS - do you want me to open a separate issue for that?

**Comment by phlummox** [comment]
Ah - from a quick look at the code, it seems like what's necessary is to edit `api.js`, and replace

~~~
         server = restify.createServer({
             log: log,
            name: global.apiName
         });
~~~

with

~~~
         server = restify.createServer({
             log: log,
            name: global.apiName,
            certificate: fs.readFileSync('../path/to/server-cert.pem'),
            key: fs.readFileSync('../path/to/server-key.pem')
         });
~~~

or similar. Seems to be working fine.

**Comment by nero120** [comment]
Hi @phlummox, apologies for the lateness of this response. I'm currently working on `v1.1.0` of the API and will add support for enabling the service to run directly over TLS.

**Comment by phlummox** [comment]
No worries, thanks for that. I quite understand being busy, I've had to put the work I was doing with the server on hold for a bit anyway.

**Comment by nero120** [comment]
Fixed in 62f4dc6868a9ac9623464b4eb97115977df5351d.
